src/components/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export function Button({ children }) {
+  return (
+    <button className="bg-gradient-to-b from-violet to-violet-dark font-bold py-3 px-8 rounded-full w-max">
```

Comment:
What does `w-max` do? 🤔

Reply:
width: max-content;

Reply:
What if you don't give it any max-width at all? Buttons should behave like you want by default, I think.

Reply:
It worked when I added parents to the button and made it inline-block. Thank you!